app/controllers/admin/depositors_search_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Admin
+  # Controller to allow searching for depositor SUNet IDs given druids
+  class DepositorsSearchController < Admin::ApplicationController
```

Comment:
My first challenge when thinking about adding an admin function is how best to set it up, in terms of routes and controller actions, in line with the sort of RESTful resource-oriented architecture that Rails makes so easy. I'm not sure I like what I decided upon here. Do our current admin functions follow any set of practices in this area? If so, I'm all ears. (I could just pick a random one and cargo-cult my way to implementation, but I think it's more important that we developers **grok** how we're mapping diff admin functions in H3.)

Reply:
Yeah, this is tricky. My general philosophy has been to follow Rails conventions as long as it seemed like a natural sense and would make sense to other Rails devs.

Here's an example of another search controller: https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/controllers/admin/druid_search_controller.rb

My thought is that `new` is appropriate for rendering a search form but none of the other Rails routes are a good fit for performing the search.